<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VoiceSettings_panel</class>
 <widget class="QWidget" name="VoiceSettings_panel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>400</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>400</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>400</width>
    <height>400</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="VoiceSetting_panel_background">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>400</width>
     <height>400</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>400</width>
     <height>400</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>400</width>
     <height>400</height>
    </size>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="resources_file.qrc">:/game_resources/voiceforce_settings_background.png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="Megaphone_image">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>180</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>40</width>
     <height>40</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>40</width>
     <height>40</height>
    </size>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="resources_file.qrc">:/game_resources/Megaphone(1).png</pixmap>
   </property>
  </widget>
  <widget class="QProgressBar" name="Voiceforce_bar">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>180</y>
     <width>150</width>
     <height>40</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>150</width>
     <height>40</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>150</width>
     <height>40</height>
    </size>
   </property>
   <property name="value">
    <number>24</number>
   </property>
  </widget>
  <widget class="QLineEdit" name="Voiceforce_number">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>180</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>40</width>
     <height>40</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>40</width>
     <height>40</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="OK_pushbutton">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>240</y>
     <width>80</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
     <italic>true</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="resources_file.qrc"/>
 </resources>
 <connections/>
</ui>
